src/docker.cc

```diff
@@ -169,6 +170,13 @@ json::value DockerReader::QueryDocker(const std::string& path) const
   }
   try {
     http::local_client::response response = client.get(request);
+    if (status(response) != 200) {
```

Comment:
Just as a note, I'm fine with just a 200 here as you have it.

Reply:
Changed for consistency.